src/main/java/org/sonar/squidbridge/annotations/SqaleLinearRemediation.java

```diff
@@ -24,6 +24,16 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
+ * Since SQ 5.6 LTS, SQALE is not used in our model.
+ * Remediation cost should be registered manually directly when registering a rule, using the following methods from SQ 6.7 LTS API:
+ * <ul>
+ * <li>org.sonar.api.server.rule.RulesDefinition.NewRule.setDebtRemediationFunction(DebtRemediationFunction)</li>
+ * <li>org.sonar.api.server.rule.RulesDefinition.NewRule.setGapDescription(String)</li>
+ * </ul>
+ */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface SqaleLinearRemediation {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlpmaf12pdu-6jE_&open=AWHXxlpmaf12pdu-6jE_)

[ISSUE_KEY]: AWHXxlpmaf12pdu-6jE_